Libraries/PeriphDrivers/Source/SFCC/sfcc_reva.c

```diff
@@ -0,0 +1,80 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2022-2023 Maxim Integrated Products, Inc. All Rights Reserved.
+ * (now owned by Analog Devices, Inc.),
+ * Copyright (C) 2023 Analog Devices, Inc. All Rights Reserved. This software
+ * is proprietary to Analog Devices, Inc. and its licensors.
```

Comment:
Since this is a new file, lines 3-6 will need to change to:
```
 * Copyright (C) 2024 Analog Devices, Inc. All Rights Reserved. This software
 * is proprietary to Analog Devices, Inc. and its licensors.
```

Reply:
Changed.